imagebuildah/build.go

```diff
@@ -826,6 +826,18 @@ func (b *Executor) Execute(ctx context.Context, stage imagebuilder.Stage) error
 			err     error
 			imgID   string
 		)
+
+		b.copyFrom = ""
+		// Check if --from exists in the step command of COPY or ADD
+		// If it exists, assing b.copyfrom that value
+		for _, n := range step.Flags {
+			if strings.Contains(n, "--from") && (step.Command == "copy" || step.Command == "add") {
+				arr := strings.Split(n, "=")
+				b.copyFrom = b.named[arr[1]].mountPoint
+				break
+			}
+		}
+
```

Comment:
Is this happy if you have a Dockerfile with multiple 'COPY --from={image}' lines like:

```
FROM scratch as myname
COPY Dockerfile.name /

FROM scratch as myname2
COPY Dockerfile.index /

FROM scratch
COPY --from=myname /Dockerfile.name /Dockerfile.name
COPY --from=myname2 /Dockerfile.index /Dockerfile.index
```

Reply:
yup, that is the what the test Dockerfile looks like.

Reply:
I see, I missed this logic is within the loop for the child, you get a new b.copyFrom with each child, I was thinking it was only one piece of memory for all.  This works.